uint8 Local_Hour_offset
uint8 Local_Minute_offset
uint16 Year
float64 Day
uint8 Month
uint8 Hours
uint8 Minutes
float64 Seconds
